cranelift/codegen/src/legalizer/globalvalue.rs

```diff
@@ -63,6 +63,22 @@ fn vmctx_addr(inst: ir::Inst, func: &mut ir::Function) {
     func.dfg.clear_results(inst);
     func.dfg.change_to_alias(result, vmctx);
     func.layout.remove_inst(inst);
+
+    crate::trace!(
+        "expanding vmctx gv {}: fact = {:?}, vmctx arg = {}",
+        global_value,
+        func.global_value_facts[global_value],
+        vmctx
+    );
+
```

Comment:
Not 100% sure we want to leave this trace in.

Reply:
Removed!